spec/wor/test_services/some_service_head_succesful_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'spec_helper'
+require_relative 'some_service'
+require_relative '../../support/mocks/responses.rb'
+
+# rubocop:disable BlockLength
+describe SomeService do
+  describe 'head_successful' do
+    # Only testing get method as Wor::Requests::Base http defined methods will
+    # only defer when validating received opts. Then,
+    # Wor::Requests::Base.request call will behave the same for all of them.
+    describe 'get' do
+      include_context 'STUB: get_head_successful'
+
+      context 'when calling with attempting_to field defined' do
+        context 'when calling without block' do
+          subject(:service) { described_class.new }
+
+          before do
+            Spy.spy(service.logger, 'info')
+            @response = service.method_with_log_without_block
+          end
+
+          it 'raises no exception' do
+            expect { @response }.not_to raise_error
+          end
+
+          it 'logger.info has been called twice' do
+            expect(service.logger.spied_info_counter).to be(2)
+          end
+        end
+
+        context 'when calling with block' do
+          subject(:service) { described_class.new }
```

Comment:
You are repeating the subject here and in the above context, move this subject into the parent context `context 'when calling with attempting_to field defined'` of both:
- `context 'when calling without block' do `
- `context 'when calling with block'`